### ISSUE TITLE
What data format is "updated"?

### ISSUE THREAD

**Issue by BjoernPetersen** [FelixGail/Musicbot-registry#3]
The docs for the `GET` endpoint say that the `updated` property is a "number", but in the example JSON it's actually a string:

```
"updated": "number; milliseconds since registration/update"
```

Will updated be a JSON number or a number in a string?

**Comment by BjoernPetersen** [comment]
P.S. having a JSON schema or even a full OpenAPI spec would probably help here.

**Comment by FelixGail** [comment]
>P.S. having a JSON schema or even a full OpenAPI spec would probably help here.

closed by 2b4797bcc81a384694c3f2372c7aac4f48e168f9
